### ISSUE TITLE
Can't authenticate

### ISSUE THREAD

**Issue by lyubomyr-shaydariv** [dzhioev/vk_api_auth#3]
I'm trying to export messages from my VK account using [this Python script](https://github.com/coldmind/vk-dialogue-export.py). It doesn't work due to authentication issues. I also tried to run `fetch_photos.py` from this repository. This is what I get in both cases:

```
Traceback (most recent call last):
  File "./fetch_photos.py", line 49, in <module>
    token, user_id = vk_auth.auth(email, password, client_id, "photos")
  File "/home/l/win/repo/git/vk_api_auth/vk_auth.py", line 97, in auth
    raise RuntimeError("Expected success here")
RuntimeError: Expected success here
```

Is your authentication script still compatible with VK authentication? Or am I missing something to run both message exporter and gallery downloader example (I'm not a Python developer)?

My environment:
- Windows XP + Python 2.7.x (native)
- Windows 7 + Cygwin + Python 2.7.3 (from Cygwin package)

Thanks!

**Comment by dzhioev** [comment]
Is your account set up to use two-factor authentication? Two-factor authentication is not supported, even if you use device-specific password.

**Comment by lyubomyr-shaydariv** [comment]
Nope, the two-factor authentication is not enabled for my account.

**Comment by dzhioev** [comment]
Can you please run the script from branch named 'debug_print' and send output to me? I've added several log messages that'll probably help.

**Comment by lyubomyr-shaydariv** [comment]
I've just pulled your recent changes and ran the script from the branch. After the brief inspection of the script output I figured out that VK authentication reports for invalid credentials. Rechecked the bash script that contained credentials and then found a typo (`l` instead of `1`). I'm very sorry for disturbing you and misrepresenting the actual things. The message exporter now works fine too. Thanks loads for the branch that revealed a stupid typo! :)
